cinema_service/settings.py

```diff
@@ -86,8 +88,12 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": os.environ["POSTGRES_DB"],
+        "USER": os.environ["POSTGRES_USER"],
+        "PASSWORD": os.environ["POSTGRES_PASSWORD"],
+        "HOST": os.environ["POSTGRES_HOST"],
+        "PORT": os.environ["POSTGRES_PORT"],
```

Comment:
It's a good practice to handle the case where the environment variables for the database settings are not set. You can do this by providing a default value in the os.environ.get() method. For example: os.environ.get('POSTGRES_DB', 'default_db_name').